modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -63,6 +63,86 @@ definition {
 		}
 	}
 
+	@description = "LPS-180358 - Verify that the status follow the correct workflow after a finance user request more info"
+	@priority = 5
+	test CanFinanceUserVerifyStatusAfterRequestMoreInfo {
+		task ("Given that a new organization was created with employee user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+
+			EVPUtils.gotoEVP();
+
+			EVPOrganization.goToCreateOrganization(
+				button = "Create an Organization",
+				navItem = "Organization");
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Banking Info",
+				city = "City",
+				contactEmail = "Contact Email",
+				contactName = "Contact Name",
+				contactPhone = "Contact Phone",
+				country = "Country",
+				done = "true",
+				organizationName = "Organization Test",
+				organizationSite = "Social Media Link",
+				smallDescription = "Small Description",
+				state = "State",
+				street = "Street",
+				taxId = "Tax Id",
+				zipCode = "Zip");
+		}
+
+		task ("When the Manager approved an organization") {
+			User.logoutAndLoginPG(
```

Comment:
I think you can change this task for: `"And this organization is with 'Awaiting finance Approval' status "`